src/polynomial.h

```diff
@@ -131,9 +131,9 @@ template <typename T>
 void Polynomial<T>::copy(Polynomial<T>* src)
```

Comment:
Why do we have a copy function instead of overloading `operator=` or a copy constructor?